### ISSUE TITLE
Move docs from wiki to Docusaurus - 2

### ISSUE THREAD

**Issue by gaurivn** [anitab-org/mentorship-backend#748]
**Is your feature request related to a problem? Please describe.**
As of now, the whole documentation for mentorship-backend is in [the wiki](https://github.com/anitab-org/mentorship-backend/wiki).

**Describe the solution you'd like**
We will now move to Docusaurus. This issue is a continuation of Issue #747. 

**Describe alternatives you've considered** 
  Docusaurus

**Additional context**
**TODO:**
- [ ] Remove any dependencies for the old docs. (sphinx for example)
- [ ] Convert all the markdown files from wiki and put it in docs

**Comment by vaibhavarora102** [comment]
can i work on this in open source hack

**Comment by vaibhavarora102** [comment]
Hi, can you pls assign this issue to me as its already 1st of September and I guess OSH has started
Thanks

**Comment by Abjcodes** [comment]
I would like to work on this issue as a part of OSH. Can you assign me this issue? Thank You.

**Comment by gaurivn** [comment]
@vaibhavarora102, @Abjcodes, you both have been assigned one other OSH issue each. Pls send a PR for those, once that is done, you can claim available issues.

**Comment by vaibhavarora102** [comment]
I had sent an PR now kindly assign me this

**Comment by Abjcodes** [comment]
I had already made one PR. Is it possible to assign this issue to me now?

**Comment by gaurivn** [comment]
> I had sent an PR now kindly assign me this

For #747, no

**Comment by gaurivn** [comment]
> I had already made one PR. Is it possible to assign this issue to me now?

You haven't sent a PR for #781 right

**Comment by vaibhavarora102** [comment]
1 had done for #733 and #1028

**Comment by Abjcodes** [comment]
> > I had already made one PR. Is it possible to assign this issue to me now?
> 
> You haven't sent a PR for #781 right

I have submitted PR for #781 by now.

**Comment by gaurivn** [comment]
> > I had sent an PR now kindly assign me this
> 
> For #747, no

@vaibhavarora102, you had not sent PR for #747 that I assigned you the previous day.

**Comment by gaurivn** [comment]
> > > I had already made one PR. Is it possible to assign this issue to me now?
> > 
> > 
> > You haven't sent a PR for #781 right
> 
> I have submitted PR for #781 by now.

Okay assigning you @Abjcodes

**Comment by Abjcodes** [comment]
@gaurivn Thank you for assigning me this issue. If you may, can you please elaborate what should be done because I am not able to understand context.

**Comment by Anmollenka** [comment]
I would like to work on this issue.Can you please assign it to me whwn it is free??

**Comment by vaibhavarora102** [comment]
Sry but I am not getting the way to use docusauras

**Comment by gaurivn** [comment]
> Sry but I am not getting the way to use docusauras

@vaibhavarora102, I didn't get you, like if you don't want to continue working on an issue please do comment under the respective issue and I'll unassign you and you can work on other issues.

**Comment by gaurivn** [comment]
> @gaurivn Thank you for assigning me this issue. If you may, can you please elaborate what should be done because I am not able to understand context.

I'll get back to you today, this issue is a continuation of #747

**Comment by vaibhavarora102** [comment]
okay thanks

**Comment by gaurivn** [comment]
@Abjcodes, how's the progress on this issue

**Comment by Abjcodes** [comment]
> > @gaurivn Thank you for assigning me this issue. If you may, can you please elaborate what should be done because I am not able to understand context.
> 
> I'll get back to you today, this issue is a continuation of #747

@gaurivn I didn't get any feedback till now. Can you elaborate this issue because I am finding it hard to understand the context.

**Comment by SanketDG** [comment]
From what I understand,

it's basically to improve whatever's there and also figure out a deployment solution for the site.

**Comment by Abjcodes** [comment]
@gaurivn @SanketDG Can you check the status of #747? I can only complete this when #747 is closed.

**Comment by Abjcodes** [comment]
@gaurivn @SanketDG Any updates?

**Comment by SanketDG** [comment]
I am sorry, that PR will get attention when the other reviewers will get to it.

You can look at other issues meanwhile.

Also, you can create your working branch from #841 and add commits from there, so when that PR is finally merged, you will only have to rebase your PR.

**Comment by devkapilbansal** [comment]
`On hold` until #747 is completed and merged

**Comment by devkapilbansal** [comment]
@isabelcosta This issue was supposed to be a continuation of #747 But  all necessary changes are taken in consideration in first issue only.
Closing it as all necessary changes have been covered in #841 itself.
